File: day06.jl
answers = open("customs_answers.txt") do f
    map(row -> split(row, "\n"), split(read(f, String), "\n\n"))
end

println(sum(map(answer -> length(union(Set.(answer)...)), answers)))

println(sum(map(answer -> length(intersect(Set.(answer)...)), answers)))

File: day06.py
with open('customs_answers.txt') as f:
    answers = list(map(lambda row: row.split("\n"), f.read().split("\n\n")))

print(sum(map(lambda answer: len(set().union(*answer))
, answers)))

print(sum(map(lambda answer: len(set(answer[0]).intersection(*answer)), answers)))
